crates/tombi-formatter/src/formatter.rs

```diff
@@ -45,11 +45,13 @@ impl<'a> Formatter<'a> {
 
     /// Format a TOML document and return the result as a string
     pub async fn format(mut self, source: &str) -> Result<String, Vec<Diagnostic>> {
+        // println!("{:?}", Instant::now());
+
         let source_schema = if let Some(parsed) =
             tombi_parser::parse_document_header_comments(source).cast::<tombi_ast::Root>()
         {
             self.schema_store
-                .try_get_source_schema_from_ast(&parsed.tree(), self.source_url_or_path)
+                .build_source_schema_from_ast(&parsed.tree(), self.source_url_or_path)
```

Comment:
Return `Feature<Result<... >>`, so the original function name is fine.

Reply:
In my opinion, the `try_` prefix might not be strictly necessary because the function returning a `Result` already clearly indicates that it's fallible.
Typically, `try_` prefixes are used to distinguish from variants that panic or block instead of returning a `Result`. However, since that's not the case here, removing the prefix might enhance readability.

Of course, I will follow the project's naming conventions, but I believe this point is worth reconsidering.

Regarding the term `build`, I agree it might indeed be unclear. However, since the function performs relatively complex operations to construct a `SourceSchema`, the nuance might be slightly different from simply "get" something. Perhaps something like `resolve_` could work better?

Reply:
panic will be marked with `unwrap_` or `expect_`.  `try_` is correct in the sense that it returns a Result.

Now, `resolve_` is a good suggestion.  This is a method name that reminds us of “Result,” let's change it to `result_`!

However, I would expect the PR to be split into smaller pieces.

There are issues in this PR that cannot be decided at once, such as Url.

Reply:
Did you mean `resolve_`, or would you prefer naming it `result_` to indicate it returns Result?

Reply:
I mean to name it `resolve_`.